docs-kits/kits/Connector Kit/Adoption View/policies-in-catena.md

```diff
@@ -0,0 +1,31 @@
+---
+sidebar_position: 3
+description: 'Conventions and behavior specific to Catena'
+title: Policies in Catena-X
+id: connector_kit_adoption_view_policies_cx
+---
+
+## Data Sovereignty in Catena-X
+
+In Catena-X, there's a set of conventions with regard to how Providers structure their `odrl:Offer`s. They are however
+also relevant for Enablement Service Providers building components enabling connectivity to the Dataspace (as specified
+in CX-0018).
+
+As mentioned in the primer on policies, Providers and Consumers must have a common
+understanding of the meaning and consequences of `odrl:Offers` and, on a more granular level, their `odrl:Constraints`.
+That's why there is a set of predefined `odrl:Constraints` - all of which have to be [accepted explicitly](working-with-policies.md#consumer-side-odrloffer-in-a-contractrequestmessage)
+and some [checked against a Consumer's VP](working-with-policies.md#provider-side-checking-a-consumers-verifiable-presentation) additionally.
+They are formalized in the Catena-X ODRL profile (to be published soon) which extends the regular [ODRL vocabulary](https://www.w3.org/TR/odrl-vocab/).
+
+Here's a non-normative overview:
```

Comment:
I think I could copy over the sections from what we have in Industry core.

Reply:
Agree, will prepare a separate PR for this.

Reply:
Proposing to remove all general content on policies from the IC Kit [1] and move it here. Commit to this PR will follow shortly.

[1] https://github.com/catenax-ng/eclipse-tractusx.github.io/pull/74

Reply:
the referenced PR deletes content from IC, but is there also one that adds it here?

Reply:
Has (as announced) now been pushed. Please excuse the 15 Mins of delay 🙄